packages/ses/src/compartment-shim.js

```diff
@@ -168,9 +302,43 @@ export const makeCompartmentConstructor = (
       transforms = [],
       __shimTransforms__ = [],
       globalLexicals = {},
+      resolveHook,
+      importHook,
+      moduleMapHook,
     } = options;
     const globalTransforms = [...transforms, ...__shimTransforms__];
 
+    // Map<FullSpecifier, ModuleCompartmentRecord>
+    const moduleRecords = new Map();
```

Comment:
Can we use properjs typings in more places?

```suggestion
    /** @type{Map<FullSpecifier, ModuleCompartmentRecord>} */
    const moduleRecords = new Map();
```

Reply:
I would like to do a full diff just for TS checking.